largest_palindrome_product.js

```diff
@@ -5,18 +5,49 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function (digits) {
   var factor_0 = 0;
   var factor_1 = 0;
+  var dummyfactor0 = 0;
+  var dummyfactor1 = 0
   var palindromeNumber = 0;
+  let product = 0;
+  let max = 0;
 
   // do your work here
 
+  function palindromeCheck(input) {
+    var palindrome = parseInt(input.toString().split("").reverse().join(""));
 
+    if (palindrome === input) return true;
+    else return false;
+  }
 
+  function findPalindrome() {
```

Comment:
Even this doesn't need to be in a function, you could pull all the contents of this function out into the function above since you're just calling this function right before the return. It saves you from creating an extra function.